tools/istio-iptables/pkg/cmd/testdata/dns-uid-gid.golden

```diff
@@ -35,4 +35,41 @@ iptables -t raw -A OUTPUT -p udp --sport 15053 -m owner --gid-owner 1 -j CT --zo
 iptables -t raw -A OUTPUT -p udp --dport 53 -m owner --gid-owner 2 -j CT --zone 1
 iptables -t raw -A OUTPUT -p udp --sport 15053 -m owner --gid-owner 2 -j CT --zone 2
 iptables -t raw -A OUTPUT -p udp --dport 53 -d 127.0.0.53/32 -j CT --zone 2
-iptables -t raw -A PREROUTING -p udp --sport 53 -d 127.0.0.53/32 -j CT --zone 1
\ No newline at end of file
+iptables -t raw -A PREROUTING -p udp --sport 53 -d 127.0.0.53/32 -j CT --zone 1
+ip6tables -t nat -N ISTIO_INBOUND
+ip6tables -t nat -N ISTIO_REDIRECT
+ip6tables -t nat -N ISTIO_IN_REDIRECT
+ip6tables -t nat -N ISTIO_OUTPUT
+ip6tables -t nat -A ISTIO_INBOUND -p tcp --dport 15008 -j RETURN
+ip6tables -t nat -A ISTIO_REDIRECT -p tcp -j REDIRECT --to-ports 15001
+ip6tables -t nat -A ISTIO_IN_REDIRECT -p tcp -j REDIRECT --to-ports 15006
+ip6tables -t nat -A OUTPUT -p tcp -j ISTIO_OUTPUT
+ip6tables -t nat -A ISTIO_OUTPUT -o lo -s ::6/128 -j RETURN
+ip6tables -t nat -A ISTIO_OUTPUT -o lo ! -d ::1/128 -p tcp ! --dport 53 -m owner --uid-owner 3 -j ISTIO_IN_REDIRECT
+ip6tables -t nat -A ISTIO_OUTPUT -o lo -p tcp ! --dport 53 -m owner ! --uid-owner 3 -j RETURN
+ip6tables -t nat -A ISTIO_OUTPUT -m owner --uid-owner 3 -j RETURN
+ip6tables -t nat -A ISTIO_OUTPUT -o lo ! -d ::1/128 -p tcp ! --dport 53 -m owner --uid-owner 4 -j ISTIO_IN_REDIRECT
+ip6tables -t nat -A ISTIO_OUTPUT -o lo -p tcp ! --dport 53 -m owner ! --uid-owner 4 -j RETURN
+ip6tables -t nat -A ISTIO_OUTPUT -m owner --uid-owner 4 -j RETURN
+ip6tables -t nat -A ISTIO_OUTPUT -o lo ! -d ::1/128 -m owner --gid-owner 1 -j ISTIO_IN_REDIRECT
+ip6tables -t nat -A ISTIO_OUTPUT -o lo -p tcp ! --dport 53 -m owner ! --gid-owner 1 -j RETURN
+ip6tables -t nat -A ISTIO_OUTPUT -m owner --gid-owner 1 -j RETURN
+ip6tables -t nat -A ISTIO_OUTPUT -o lo ! -d ::1/128 -m owner --gid-owner 2 -j ISTIO_IN_REDIRECT
+ip6tables -t nat -A ISTIO_OUTPUT -o lo -p tcp ! --dport 53 -m owner ! --gid-owner 2 -j RETURN
+ip6tables -t nat -A ISTIO_OUTPUT -m owner --gid-owner 2 -j RETURN
+ip6tables -t nat -A ISTIO_OUTPUT -d ::1/128 -j RETURN
```

Comment:
Did a comparison between `iptables` and `ip6tables` and the only difference is iptables has `I noticed that iptables has `iptables -t nat -A ISTIO_OUTPUT -p tcp --dport 53 -d 127.0.0.53/32 -j REDIRECT --to-ports 15053` where that is missing from ipv6.

Since pilot-agent runs a local dns server on that port, would we want to extend it to 15053? I'd see that as new feature that would then be pushed back to iptables.

Reply:
This is actually working as intended, just the test is bad. We only match traffic destined to the nameserver in resolv.conf - in this case didn't update the test to include a ipv6 server